### ISSUE TITLE
Skynet/Sia Updates

### ISSUE THREAD

**Issue by pcfreak30** [ethereum/ethereum-org-website#11329]
### Describe the bug

The Skynet ecosystem (built on Sia) failed Q4 2022, and the Sia community has been rebuilding.

New projects are being built to replace Skynet made by the community, but at this time, References to Skynet should just be replaced with Sia.

Content that needs fixing is [ethereum/ethereum-org-website/src/content/developers/docs/storage/index.md](https://github.com/ethereum/ethereum-org-website/blob/10220687fb0699b992c738d72bf73d72d0a5f3bf/src/content/developers/docs/storage/index.md)

### To reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

N/A

### Would you like to work on this issue?

- [X] Yes
- [ ] No

**Comment by wackerow** [comment]
Thanks @pcfreak30! Sorry for delay here, but if you're still interested in working on this I've gone ahead and assigned you. Looks like `https://siasky.net/` is no longer an active link, I suppose we're replace with https://sia.tech/? PR welcome! 🙏

**Comment by corwintines** [comment]
@pcfreak30 just bumping to see if this is something you were still interested in adding. 

If not, will unassign and open up for someone else to add.

**Comment by pcfreak30** [comment]
@corwintines Sorry for the delay. I have been communicating with the foundation of my Sia community about this and was awaiting their response in how they wanted to proceed.

They gave me the go ahead as a community member (I am NOT part of the foundation), to continue to tackle this for the community.

Kudos.

**Comment by Wachira3712** [comment]
สวัสดีครับ ตอบ mail น่ะครับ จากคำถามชี้แนะ ผมตอบว่า ใช่ ครับ  ใช่
คือดำเนินการเลยครับ


Tel. +66 957 230 420
Wachira Suporn.test_one
Email. ***@***.***


เมื่อ จ. 12 ส.ค. 2024 เวลา 15:06 github-actions[bot] <
***@***.***> เขียนว่า:

**Comment by Wachira3712** [comment]
@pcfreak30  ขแบคุณมาก ๆ เลยน่ะครับ

**Comment by pcfreak30** [comment]
> This issue is stale because it has been open 30 days with no activity.

Bump. Still alive.

**Comment by pcfreak30** [comment]
> @pcfreak30 Thanks for the bump... So is there something blocking this right now?

no, just priorities. will get to this soon. thanks.

**Comment by pcfreak30** [comment]
still active

**Comment by pcfreak30** [comment]
still active, looking to handle this in November.

**Comment by pcfreak30** [comment]
still active

**Comment by pcfreak30** [comment]
still active

**Comment by pcfreak30** [comment]
still active

**Comment by pcfreak30** [comment]
still active

**Comment by pcfreak30** [comment]
still active

**Comment by anushkasomani** [comment]
> still active
> 
> 

Can I work on this issue??

**Comment by pcfreak30** [comment]
> > still active
> 
> Can I work on this issue??

No. I will be getting to this soon if the foundation doesn't. I have just have had a lot higher priorities to deal with.
